engine/render/src/render/font.h

```diff
@@ -33,8 +33,10 @@ namespace dmRender
     {
         float m_MaxAscent;
         float m_MaxDescent;
-        uint32_t m_MaxWidth;   // The widest one in terms of texels
-        uint32_t m_MaxHeight;  // The tallest one in terms of texels
+        float m_MaxWidth;           // The widest glyph
+        float m_MaxHeight;          // The tallest glyph
+        uint16_t m_ImageMaxWidth;   // The widest glyph (in texels)
+        uint16_t m_ImageMaxHeight;  // The tallest glyph (in texels)
```

Comment:
When doing tex metrics in the renderer, we need both the glyph size, and the glyph bitmap size